tests/unit/backend/controllers/email-submissions.server.controller.spec.js

```diff
@@ -1068,89 +1058,6 @@ describe('Email Submissions Controller', () => {
       prepareSubmissionThenCompare(expected, done)
     })
 
-    it('excludes field if isVisible is false for autoReplyData', (done) => {
```

Comment:
why are all these tests deleted?

Reply:
Thanks for highlighting. I removed them initially as they were invalid since they were inputting responses to hidden fields, but on review, I've instead fixed the faulty values